fmf/utils.py

```diff
@@ -782,37 +783,28 @@ def run(command, cwd=None, check_exit_code=True, env=None):
 #  Convert dict to yaml
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-# Special hack to store multiline text with the '|' style
-# See https://stackoverflow.com/questions/45004464/
-yaml.SafeDumper.orig_represent_str = yaml.SafeDumper.represent_str
-
-
-def repr_str(dumper, data):
-    if '\n' in data:
-        return dumper.represent_scalar(
-            u'tag:yaml.org,2002:str', data, style='|')
-    return dumper.orig_represent_str(data)
+def dict_to_yaml(data, width=None, sort=False):
+    """ Convert dictionary into yaml """
+    output = StringIO()
 
+    # Set formatting options
+    yaml = YAML()
+    yaml.indent(mapping=4, sequence=4, offset=2)
+    yaml.default_flow_style = False
+    yaml.allow_unicode = True
+    yaml.encoding = 'utf-8'
+    yaml.width = width
 
-yaml.add_representer(str, repr_str, Dumper=yaml.SafeDumper)
+    # Make sure that multiline strings keep the formatting
+    data = copy.deepcopy(data)
```

Comment:
this line here is to avoid `YAML` referencing abilitiers?

Reply:
The `walk_tree()` method used on the next line modifies all strings which have multiple lines. Although this change to `LiteralScalarString` should not affect the string-related methods and should be relatively ok, I think it is safer not to modify provided `data` at all.

Reply:
okay, thanks for explanation